force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +118,26 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    public static Map<Id, Task> getTaskDueNextWeek () {
     //  The body of the method
-    //}
+
+    Map<Id, Task> taskIdToTask = new Map<Id, Task>();
+
+    Date todaysDate = Date.today();
+
+    List<Task> taskDueWithin7Days = [SELECT Id, Subject, ActivityDate FROM Task
+     WHERE ActivityDate >= :todaysDate  AND ActivityDate <= :todaysDate.addDays(7)];
+    
+
+    for(Task t : taskDueWithin7Days){
```

Comment:
Try to use the  [date literal](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_dateformats.htm)  

Try to avoid single letter variables